blockball-bukkit-plugin/src/main/java/com/github/shynixn/blockball/bukkit/BlockBallDependencyInjectionBinder.kt

```diff
@@ -117,7 +117,7 @@ class BlockBallDependencyInjectionBinder(private val plugin: BlockBallPlugin) :
         bind(DependencyBossBarApiService::class.java).to(DependencyBossBarApiServiceImpl::class.java)
             .`in`(Scopes.SINGLETON)
         bind(DependencyService::class.java).to(DependencyServiceImpl::class.java).`in`(Scopes.SINGLETON)
-        bind(PackageService::class.java).to(PackageServiceImpl::class.java).`in`(Scopes.SINGLETON)
+        bind(ProxyService::class.java).to(ProxyService::class.java).`in`(Scopes.SINGLETON)
```

Comment:
ProxyService is now declared twice in this file. Please remove the line.

Reply:
Also, the travis CI build seems to be failing because the there is a compilation issue of the test classes.

Reply:
Changed, how can I check the travis step?

Reply:
You can locally execute ``gradle build`` to see where it fails. For travis CI, the pull request states are visible here:

https://travis-ci.org/github/Shynixn/BlockBall/pull_requests

Reply:
Now the travis job complete without errors